tools/Profiler/NanoProfiler/.editorconfig.txt

```diff
@@ -0,0 +1,276 @@
+# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
This shouldn't be in the PR, please remove and maybe add it to gitignore please.

Reply:
This file allows to format code, normally we have to set it  for all projects.. Goal: each developer should use the same code formatter

Reply:
I'm familiar with editorconfig file. 😉 

The project already has a code style defined. And stylecop rules set. If you're keen to have these setup, OK to ad stylecop on a separate PR.

Reply:
ok

Reply:
@Ghislain1 this is been here for a while... just wanted to know if you'll be addressing the above comment. Thanks! :)